CHANGELOG.md

```diff
@@ -72,6 +72,7 @@
 
 - [CVE-2023-26136] Add resolution for tough-cookie to ^4.1.3 ([#889](https://github.com/opensearch-project/oui/pull/889))
 - [CVE-2023-26115] Bump word-wrap from 1.2.3 to 1.2.4 ([#891](https://github.com/opensearch-project/oui/pull/891))
+- Remove Internet Explorer specific code ([#890](https://github.com/opensearch-project/oui/pull/890))
```

Comment:
This should be moved to `unreleased` section before merging. This is an artifact of how updating the changelog was done, sorry!

Reply:
done